core/app/[locale]/(default)/cart/page.tsx

```diff
@@ -20,6 +20,8 @@ interface Props {
   params: Promise<{ locale: string }>;
 }
 
+const isUsingTrailingSlashes = process.env.TRAILING_SLASH !== 'false';
```

Comment:
🍹 I would just define the URL globally here
```suggestion
const CHECKOUT_URL = process.env.TRAILING_SLASH !== 'false' ? '/checkout/' : '/checkout';
```